guides/m1x/system-requirements.md

```diff
@@ -34,7 +34,7 @@ MySQL 5.6 (Oracle or Percona)
     
 	Redis can be used for session or cache storage
 
-*   memcached
+*   <a href="{{ page.baseurl }}/config-guide/memcache/memcache.html">Memcached</a>
```

Comment:
This link doesn't actually work, I get a 404. You're already within `/guides/mx1` for this page, so it gives you the URL `/guides/mx1/<version>/config-guide/memecache/memcache.html`, which doesn't work. 

Please change to full URL for the page , without the `page.baseurl`.

Reply:
@shrielenee I have updated link. Please check it.